docs/_documentations/gettingstarted-overview.md

```diff
@@ -7,55 +7,65 @@ duration: 10 minutes
 permalink: gettingstarted-overview.html
 type: document
 ---
-# Overview: Getting Started
+# Getting Started - Overview
 
-**Getting started quickly with Codewind** You can try out Codewind by using the local configuration. For local use of Codewind, and also the pre-requisite steps for remote use of Codewind, select from:
+<div class="callout">
+	<b>Getting started quickly with Codewind</b> You can try out Codewind by using the local configuration. For local use of Codewind, and also the prerequisite steps for remote use of Codewind, select from <a href="./vsc-getting-started.html">VS Code,</a> <a href="./eclipse-getting-started.html">Eclipse,</a> or <a href="./intellij-getting-started.html">Intellij.</a>
+</div>
 
-* [VS Code](./vsc-getting-started.html)
-* [Eclipse](./eclipse-getting-started.html)
-* [IntelliJ](./intellij-getting-started.html)
-
-**Codewind as a hosted application in the cloud** If you want to go straight to using codewind as a hosted application in the cloud, select the [Eclipse Che instructions](./eclipseche-codewind-overview.html)
+<div class="callout">
+	<b>Browser-based Codewind</b> If you want to go straight to using codewind as a hosted application in the cloud, select the <a href="./eclipseche-codewind-overview.html">Eclipse Che instructions. </a>
+</div>
 
-**The different configurations of Codewind** To find out about the different ways of using Codewind - locally, remotely, or as an application hosted on the cloud - see [Codewind Architecture](./overview.html#architecture).
-
-### Which configuration do I need?
+## Which configuration do I need?
 
 Codewind supports different configurations for developing your containerized applications, so you can use Codewind with whichever development flow you choose.
 
 You can do your development work in an IDE on your desktop, or in an IDE hosted on the cloud.
 
-You can choose to build and run your application locally, or use the cloud's resources by using Codewind remotely or as an application hosted on the cloud. Codewind uses [Kubernetes](https://kubernetes.io/) container orchestration applications, including [OpenShift](https://www.openshift.com/) when building and run your Docker-containerised applications.
+You can choose to build and run your application locally, or build and run your application using the cloud's resources. 
+
+Both the remote configuration and browser-based configuration of Codewind use the cloud's resources to build and run your application. Codewind uses [Kubernetes](https://kubernetes.io/) container orchestration applications, including [OpenShift](https://www.openshift.com/), to build and run your Docker-containerised applications on the cloud.
+
+Continue to learn about each of these configurations by selecting the relevant section:
```

Comment:
I like: 
"Continue to learn more about each of these configurations by clicking the following options:"